test/DividendPayableToken.dividend.test.js

```diff
@@ -10,6 +10,17 @@ const should = require('chai')
   .use(require('chai-bignumber')(BigNumber))
   .should();
 
+var assertRevert= async function(promise){
```

Comment:
Please replaced it by 
`import EVMRevert from 'openzeppelin-solidity/test/helpers/EVMRevert'; `